<launch>

	<node pkg="lab3" type="mpc_controller.py" name="mpc_controller" output="screen" >
    <param name = "speed" type = "double" value="1"/>
    <param name = 'min_delta' type = 'double' value='-0.34'/>
    <param name = 'max_delta' type = 'double' value='0.341'/>
    <param name = 'delta_incr' type = 'double' value='0.34'/>
    <param name = 'lookahead_distance' type = 'double' value = '1.0'/>
    <param name = 'T' type = 'int' value='1'/>
    <param name = 'dt' type = 'double' value='0.01'/>
    <param name = 'compute_time' type = 'double' value='0.09'/>
    <param name = 'laser_offset' type = 'double' value='1.0'/>
    <param name = 'laser_window' type = 'int' value='20'/>
    <param name = 'bag_path' type = 'string' value='/home/car-user/racecar_ws/src/lab3/bags/full_car_plan.bag'/>

	</node>

</launch>
